src/epics/data.ts

```diff
@@ -518,8 +518,8 @@ const saveAssociationsActive: Epic = (action$, store) => action$.ofType(types.sa
     const pendingChanges = (state.view.pendingDataChanges[
         action.payload.bcNames[0]
     ] || {})
-
-    return api.associate(state.screen.screenName, bcUrl, Object.values(pendingChanges) as AssociatedItem[])
+    const params: Record<string, any> = action.payload.bcNames.length ? {_bcName: action.payload.bcNames.pop()} : {}
```

Comment:
`Array.prototype.pop` will modify the source array (action.payload.bcNames) and incorrect array will be forwarder through an action chain (e.g. bcCancelPendingChanges)

Reply:
fixed as `array[array.length - 1]`